README.md

```diff
@@ -13,3 +13,26 @@ This guide provides a uniform way to structure your RiotJS code. Making it
 * easier to cache and serve bundles of code separately.
 
 This guide is inspired by [AngularJS Style Guide](https://github.com/johnpapa/angular-styleguide) by John Papa.
+
+## Table of Contents
+
+1. [Component / Tag names](#component-tag-names)
```

Comment:
Agreed, we need a ToC. But I think it's going to be hard to order them, so let's maybe stick with an unordered list and do try to arrange them from big to small and from start of projects to finetuning a project. That way a developer can read them like a recipe.